### DISCUSSION TITLE
Loading QField projects onto iPad

### DISCUSSION THREAD

**Discussion by AK-survey** [opengisch/QField#3353] [Q&A]
Hi, just getting into testing QField for QGIS and have been testing so far on Android. However, my company also runs iPads for field use and we're trying to test it there also. How can I upload the QField project onto the iPad? I'm familiar with using iTunes (I'm running Windows 10) to upload files to apps, but don't see a way to upload to any other folders on the device. Any help is appreciated.

-David

**Comment by nirvn** [comment]
@AK-survey , I'm happy to report the latest testflight version has gained the capability to do USB transfers via iTunes; if you want to stick to manual transfers, you are now served :) If you are managing a fleet of iPads in your company, I'd strongly advise to give QFieldCloud a look. It enables easy remote deployment and management of projects, without the need for your iPads to be physically back at your office.
